packages/create-app/templates/default-app/package.json.hbs

```diff
@@ -29,9 +30,10 @@
   },
   "devDependencies": {
     "@backstage/cli": "^{{version '@backstage/cli'}}",
-    "@spotify/prettier-config": "^7.0.0",
+    "concurrently": "^6.0.0",
     "lerna": "^4.0.0",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "@spotify/prettier-config": "^7.0.0"
```

Comment:
heh, why was this moved :) no biggie, just curious

Reply:
so they are alphabetical? happy to move it back up 😁

Reply:
yepyep, doesn't make a difference except next time someone does a `yarn add` the script will re-sort so the diffs are unnecessarily larger 🧶

Reply:
ah - sorry, didn't know `@` auto sorted earlier in the path. tested offline, and learned something. fixed!